tests/derive/help.rs

```diff
@@ -395,7 +395,7 @@ Arguments:
           Argument help
 
           Possible values:
-          - foo: Foo help
+          - foo: Foo help (alias: foozle)
```

Comment:
So if no help is specified, it will be `foo: (alias: foozle)`.  That feels a little awkward

Reply:
@epage Formatting suggestions welcome. I considered making it part of `foo` instead, but then it'd be taken into account for indentation.

Reply:
Let's start with this.  This is making me want to revisit next_line_help's behavior and, in doing so, I hope to simplify cases like this.  I just don't think looking at that should block this moving forward.